browser/js/signup/signup.html

```diff
@@ -1,42 +1,55 @@
-<section id='signup-form'>
-    <h1>Sign Up</h1>
-    <div>
-        <alert type="danger" ng-show="error">
-            {{ error }}
-            <br>
-        </alert>
-        <form name="myForm">
-            <div class="form-group">
-                <label>First Name:</label>
-                <input type="text" class="inputColor form-control" ng-model="user.firstName" required/>
-            </div>
-            <div class="form-group">
-                <label>Last Name:</label>
-                <input type="text" class="inputColor form-control" ng-model="user.lastName" required/>
-            </div>
-            <div class="form-group">
-                <label>Your Email:</label>
-                <input type="email" class="inputColor form-control" ng-model="user.email" required/>
-            </div>
-            <div class="form-group">
-                <label>Your Password:</label>
-                <input type="password" class="inputColor form-control" ng-model="user.password" required/>
+<div class="container">
+    <div class="row">
+        <div class="col s8 offset-s2">
+            <div class="row">
+                <h1>Sign Up</h1>
             </div>
-
             <div class="row">
-                <div class="col s6 form-group left">
-                    <label>Your zip code (for planting purposes):</label>
-                    <input type="text" class="inputColor form-control" ng-model="user.zip" required/>
-                </div>
-                <div class="col s6 form-group right">
-                    <label>Your phone number (optional for weather alerts):</label>
-                    <input type="text" class="inputColor form-control" ng-model="user.phoneNumber" required/>
-                </div>
+                <alert type="danger" ng-show="error">
+                    {{ error }}
+                    <br>
+                </alert>
             </div>
-
-            <div class="form-group">
-                <button type="button" class="btn-flat" ng-disabled="myForm.$invalid" ng-click="createUser(user)">Submit</button>
+                <form name="myForm">
```

Comment:
isn't this the same as the old signup form? why does it look like it was completely rewritten?